src/integrationTest/resources/application.yml

```diff
@@ -27,13 +27,24 @@ spring:
       max-file-size: 8MB
       max-request-size: 10MB
       resolve-lazily: true
+  cloud:
+    aws:
+      s3:
+        endpoint: http://s3.localhost.localstack.cloud:4566
```

Comment:
Should it have localstack twice in the url?

Reply:
It's localhost and localstack

Reply:
Should have gone to specsavers